presentation/common/src/test/content/Descriptor.test.ts

```diff
@@ -86,6 +87,23 @@ describe("Descriptor", () => {
       expect(descriptor).to.matchSnapshot();
     });
 
+    it("creates valid descriptorJSON from CompressedDescriptorJSON with nestedField", () => {
+      const compressedDescriptorJSON = compressDescriptorJSON(testDescriptorJSON);

```

Comment:
This test is testing not only the stuff it needs to (`fromJSON` overload with `CompressedDescriptorJSON` argument), but also `compressDescriptorJSON` function, which we don't want.

Instead, I recommend creating a `CompressedDescriptorJSON` object, deserializing `Descriptor` from it and making sure it's okay. The JSON can contain multiple different fields to avoid having two similar tests.

Reply:
Added new tests for `CompressedDescriptorJSON` and `compressDescriptorJSON`. Used `matchSnapshot` for both of them, was that the right approach?